src/v/resource_mgmt/memory_groups.h

```diff
@@ -50,4 +50,9 @@ struct memory_groups {
     static size_t recovery_max_memory() {
         return ss::memory::stats().total_memory() * .10;
     }
+
+    /// Max memory that both cloud storage uploads and read-path could use
+    static size_t tiered_storage_max_memory() {
+        return ss::memory::stats().total_memory() * .10;
```

Comment:
Previously, the percentages added up to 100%. I feel like that was intentional. Does it mean we should reduce allocation for other memory groups?

Reply:
I think it wasn't added up to 100. The `chunk_cache_max_memory` and `chunk_cache_min_memory` shouldn't be both added to the final percentage. So it was counting to 90% or to 70% (depending on which chunk cache size you pick) and now it's counting up to 100% or 80%.